### ISSUE TITLE
QUESTION CREATE API

### ISSUE THREAD

**Issue by phfontes** [Azure-Samples/azure-search-openai-demo#518]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
Windows 11

### azd version?
> run `azd version` and copy paste here.
1.1.0

### Versions
>

### Mention any other details that might be useful
Its possible to create a API of this plataform and connected a other service the chatbot that realize search de conteúdo?
Or, its possibile to connected in bot of Microsoft Teams ?
> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
Yes, @garrytrinder has built a prototype Teams bot that uses this app as an API. See the comment here:

https://github.com/Azure-Samples/azure-search-openai-demo/issues/451#issuecomment-1664023194

**Comment by pamelafox** [comment]
Closing this issue, please continue discussion on issue #451, thanks!
